src/NuGet.Core/NuGet.Protocol.Core.v3/Utility/ProtocolUtility.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using NuGet.Common;
+
+namespace NuGet.Protocol
+{
+    public class ProtocolUtility
+    {
+        /// <summary>
+        /// Set ServicePointManager.DefaultConnectionLimit
+        /// </summary>
+        public static void SetConnectionLimit()
```

Comment:
Can [NuGet.CommandLine.Program.MainCore(...)](https://github.com/NuGet/NuGet.Client/blob/4cccb13833ad29d6a0bcff055460d964f1b49cfe/src/NuGet.Clients/NuGet.CommandLine/Program.cs#L70) be updated to use this?